### ISSUE TITLE
Non quest items reset characters 

### ISSUE THREAD

**Issue by Ephreaym** [opennox/opennox#670]
Not sure what the cause is or how to circumvent is. It looks to me like an anti-cheating system. But troublesome when trying to add quest items.

**Comment by KILAH4716** [comment]
This also occurs if you try to add an illegal item (like "of the Dragon's Heart of Anti-Lightning Breastplate of the Scorpion's Bane and Regeneration"). There's obviously a check going on to ensure this is a valid item that can be found in quests as you load the game. If it's not, then _Elitas Deletus_ and your character becomes corrupted.

**Comment by dennwc** [comment]
Thank you for checking that! I didn't touch the item save/load so this might take some time.

**Comment by Ephreaym** [comment]
Maybe we don’t need to change how the game handles the saving and loading. Maybe we can identify how to mark something as obtained in quest mode? And then mark anything we want as “obtained in quest”?

**Comment by dennwc** [comment]
Yes, ideally we'd want to alter the save/load process completely, especially for the open world. But it will take time.

So for this specific issue, I'd like to at least understand why the reset happens at all and how we can (optionally) bypass it.

**Comment by Ephreaym** [comment]
I just found out that if you select the "marked" option in the map editor or through ns4 (for example: "bow.FlagsEnable(object.FlagMarked)" it will not trigger the anti-cheating system!

**Comment by Ephreaym** [comment]
Now if we can change stats/enchants it'll work great I think.

**Comment by Ephreaym** [comment]
Actually, I was just lucky with the item being legal. It didn't have anything to do with the marked option.

**Comment by dennwc** [comment]
Looks like it's happening during save file load procedure, not during saving. Save file looks valid after picking up these items. But loading it afterwards resets everything.
